kube_apiserver_metrics/README.md

```diff
@@ -14,26 +14,122 @@ The Kube_apiserver_metrics check is included in the [Datadog Agent][3] package,
 
 ### Configuration
 
-The main use case to run the kube_apiserver_metrics check is as a Cluster Level Check.
-See the documentation for [Cluster Level Checks][4].
+- The main use case to run the kube_apiserver_metrics check is as a [Cluster Level Check][4]. 
+- To collect metrics, set the following parameters and values in an [Autodiscovery][12] template. 
+- You can do this with [annotations on your service](#annotate-service), or by using a [local file](#local-file) through the Datadog Operator, Helm Chart or manually. 
+
+| Parameter         | Value                                                                 |
+|-------------------|-----------------------------------------------------------------------|
+| `<INTEGRATION_NAME>`| `["kube_apiserver_metrics"]`                                            |
+| `<INIT_CONFIG>`     | `[{}]`                                                                  |
+| `<INSTANCE_CONFIG>` | [{ `"prometheus_url": "https://%%host%%:%%port%%/metrics", "bearer_token_auth": "true"` }] |
+
+You can review all available configuration options in the [kube_apiserver_metrics.yaml][7].
+
+#### Annotate service
 You can annotate the service of your apiserver with the following:
 
+{{< tabs >}}
+{{% tab "Annotations v2 (for Datadog Agent v7.36+)" %}}
+
+```yaml
+ad.datadoghq.com/service.checks: |
```

Comment:
The original documentation was proposing to use endpoint checks instead of service check ([line 23 in the original version](https://github.com/DataDog/integrations-core/pull/18822/files#diff-18c38272846244d0c4993812ff9abf0dd4072a5ac48907dddab977d639552398L23-L25)).
I think it made sense since [a lot of metrics](https://github.com/DataDog/integrations-core/blob/master/kube_apiserver_metrics/metadata.csv) are “per instance”.

In order to use endpoint check, we would need:
```suggestion
ad.datadoghq.com/endpoints.checks: |
```

Reply:
fixed